Header header
string base_link
string distal_link
geometry_msgs/Pose[] frames
geometry_msgs/Twist[] twists
duration[] time_from_start
